src/com/b44t/messenger/DcContext.java

```diff
@@ -184,6 +195,23 @@ public void unref() {
     // helper to get/return strings from/to handleEvent()
     public native static boolean data2IsString(int event);
 
+    /**
+     * Gets a string you can show to the user with basic information about connectivity.
+     * @param context
+     * @param connectedString Usually "Connected", but when using this as the title in
+     *                        ConversationListActivity, we want to write "Delta Chat" there instead.
+     * @return
+     */
+    public static String getConnectivitySummary(Context context, int connectedString) {
```

Comment:
this is a regression, needs android specific stuff and parts of the delta chat application, reverting previous efforts to keep pure Java bindings modular and separated from the rest of the application, 

the bindings should be used as if they were an external dependency, and actually would be great if at some point they were just a line in the dependencies section in build.gradle to avoid future temptations ;)

I guess this should be added to ApplicationDcContext maybe?

Reply:
Will change that, thanks for taking care